src/components/Menu/Menu.style.ts

```diff
@@ -0,0 +1,72 @@
+import { backgroundPickerBasedOnType, colorPickerBasedOnType } from 'utils/themeFunctions';
+import { Props } from 'components/Button/Button';
+import { RequiredProperties } from 'utils/common';
+import { Theme } from 'theme';
+import { FlexDirectionProperty } from 'csstype';
+import { rem } from 'polished';
+
+/** Calculates the button specific height based on the size passed to it
+ * These sizes are specific to this button thus these are placed here and not in the config **/
+const heightBasedOnSize = (size: 'lg' | 'md' | 'sm') => {
```

Comment:
Since this function implementation is repeated in components Button, IconButton and Menu you could move this into a helper function. 
Furthermore, you could refactor "switch" with bracket notation which, i think, it is more efficient.   

example in common.ts:
```
export interface SizeConfig {
  lg: string;
  md: string;
  sm: string;
}
```

example in helpers.ts 
```
const SIZE_OPTIONS: SizeConfig = {
  lg: rem(56),
  md: rem(46),
  sm: rem(40),
};

export const heightBasedOnSize = (size: 'lg' | 'md' | 'sm'): string => SIZE_OPTIONS[size];
```